From cc2cb8496257eaf32fdce442b6784d0b4a0ec45e Mon Sep 17 00:00:00 2001
From: Minghuan Lian <Minghuan.Lian@nxp.com>
Date: Tue, 16 Aug 2016 16:48:11 +0800
Subject: [PATCH 047/388] arm64: dts: ls2080a: increase PCIe configuration
 space size

The Designware PCIe driver splits the configuration space to
two parts, one for CFG0, the other for CFG1. Each part will be
described by an iATU. The minimum size of iATU is 4K.
The patch increases the configuration size to 8K bytes to contain
two minimum iATUs.

Signed-off-by: Minghuan Lian <Minghuan.Lian@nxp.com>
[Original patch taken from SDK-V2.0-1703]
Signed-off-by: Yanjiang Jin <yanjiang.jin@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-ls2080a.dtsi | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-ls2080a.dtsi b/arch/arm64/boot/dts/freescale/fsl-ls2080a.dtsi
index f0e58f6..f94cc93 100644
--- a/arch/arm64/boot/dts/freescale/fsl-ls2080a.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-ls2080a.dtsi
@@ -406,7 +406,7 @@
 		compatible = "fsl,ls2080a-pcie", "fsl,ls2085a-pcie",
 			     "snps,dw-pcie";
 		reg = <0x00 0x03400000 0x0 0x00100000   /* controller registers */
-		       0x10 0x00000000 0x0 0x00001000>; /* configuration space */
+		       0x10 0x00000000 0x0 0x00002000>; /* configuration space */
 		reg-names = "regs", "config";
 		interrupts = <0 108 0x4>; /* aer interrupt */
 		interrupt-names = "aer";
@@ -431,7 +431,7 @@
 		compatible = "fsl,ls2080a-pcie", "fsl,ls2085a-pcie",
 			     "snps,dw-pcie";
 		reg = <0x00 0x03500000 0x0 0x00100000   /* controller registers */
-		       0x12 0x00000000 0x0 0x00001000>; /* configuration space */
+		       0x12 0x00000000 0x0 0x00002000>; /* configuration space */
 		reg-names = "regs", "config";
 		interrupts = <0 113 0x4>; /* aer interrupt */
 		interrupt-names = "aer";
@@ -456,7 +456,7 @@
 		compatible = "fsl,ls2080a-pcie", "fsl,ls2085a-pcie",
 			     "snps,dw-pcie";
 		reg = <0x00 0x03600000 0x0 0x00100000   /* controller registers */
-		       0x14 0x00000000 0x0 0x00001000>; /* configuration space */
+		       0x14 0x00000000 0x0 0x00002000>; /* configuration space */
 		reg-names = "regs", "config";
 		interrupts = <0 118 0x4>; /* aer interrupt */
 		interrupt-names = "aer";
@@ -481,7 +481,7 @@
 		compatible = "fsl,ls2080a-pcie", "fsl,ls2085a-pcie",
 			     "snps,dw-pcie";
 		reg = <0x00 0x03700000 0x0 0x00100000   /* controller registers */
-		       0x16 0x00000000 0x0 0x00001000>; /* configuration space */
+		       0x16 0x00000000 0x0 0x00002000>; /* configuration space */
 		reg-names = "regs", "config";
 		interrupts = <0 123 0x4>; /* aer interrupt */
 		interrupt-names = "aer";
-- 
2.9.3

